test/js-native-api/test_string/test_string.c

```diff
@@ -310,6 +310,39 @@ static napi_value TestPropertyKeyUtf16AutoLength(napi_env env,
                          auto_length);
 }
 
+static napi_value TestPropertyKeyUtf8(napi_env env, napi_callback_info info) {
```

Comment:
This function can be greatly simplified by simply calling the `TestOneByteImpl` function.
See the PR #50282 where you had `TestPropertyKeyUtf16` and `TestPropertyKeyUtf16AutoLength` that simply call `TestTwoByteImpl`. Here we can do the same: we should have four functions: `TestPropertyKeyUtf8`, `TestPropertyKeyUtf8AutoLength`, `TestPropertyKeyLatin1`, and `TestPropertyKeyLatin1AutoLength` that just call the `TestOneByteImpl` with the corresponding parameters.